aten/src/ATen/core/jit_type.h

```diff
@@ -1518,7 +1518,7 @@ template <typename T>
 struct getTypePtr_ final {
   static TypePtr call() {
     if (!isCustomClassRegistered<T>()) {
-      throw c10::Error("Type could not be converted to any of the known types.", "");
+      throw c10::Error(std::string("Type ") + std::string(c10::util::get_fully_qualified_type_name<T>()) + " could not be converted to any of the known types.", "");
```

Comment:
Wonder why this isn't just using `TORCH_CHECK`

Reply:
+1